src/sign.ts

```diff
@@ -98,12 +90,21 @@ export class Signer {
       `https://rekor.sigstore.dev/api/v1/log/entries/${entry.uuid}`
     );
 
-    const signedPayload: SignedPayload = {
-      base64Signature: signature,
+    const bundle: SigstoreBundle = {
+      attestationType: 'attestation/blob',
+      attestation: {
+        payloadHash: digest,
+        payloadAlgorithm: 'sha256',
+        base64Signature: signature,
+      },
       cert: b64Certificate,
```

Comment:
Do we want to align with the [current (in flight spec)](https://docs.google.com/document/d/1gucjOA_bGyRjK6TeaOI-X5GIUv8WsPzeMDMkq25Kv4Y/edit#heading=h.700g68d1uga)?
```suggestion
      certificate: b64Certificate,
```